### ISSUE TITLE
FrescoImage crash on null imageUrl

### ISSUE THREAD

**Issue by CloudyDino** [skydoves/Landscapist#62]
**Please complete the following information:**
- Library Version: landscapist-fresco v1.3.6
- Affected Device(s): Pixel 3 Emulator with Android 11.0
 
**Describe the Bug:**
Tried to make a `FrescoImage` with a null `imageUrl` and painter resource for the error argument, but I got a crash because the imageUrl was null. Happens because `LocalFrescoProvider.getFrescoImageRequest` takes a nullable imageUrl as an argument, but can't handle it correctly since `Uri` can only parse non-null urls.

```yaml
2021-09-28 13:03:10.530 22294-22294/com... E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com..., PID: 22294
    java.lang.NullPointerException: uriString
        at android.net.Uri$StringUri.<init>(Uri.java:496)
        at android.net.Uri$StringUri.<init>(Uri.java:486)
        at android.net.Uri.parse(Uri.java:458)
        at com.skydoves.landscapist.fresco.LocalFrescoProvider.getFrescoImageRequest(LocalFrescoProvider.kt:40)
        at com.skydoves.landscapist.fresco.FrescoImage__FrescoImageKt.FrescoImage(FrescoImage.kt:84)
        at com.skydoves.landscapist.fresco.FrescoImage.FrescoImage(FrescoImage.kt:1)
        at ...
```

**My Code**
```kotlin
FrescoImage(
    imageUrl = nullableUrl,
    placeHolder = painterResource(R.drawable.placeholder),
    error = painterResource(R.drawable.placeholder)
)
```

**Expected Behavior:**
Show the error image and don't crash.

**Possible Fix**
1. Don't let `LocalFrescoProvider.getFrescoImageRequest` take a nullable image URL and have a different default for the `ImageRequest` parameter in `FrescoImage`.
2. Replace `Uri.parse(imageUrl)` with `Uri.parse(imageUrl ?: "")`

**Comment by skydoves** [comment]
Hi @CloudyDino,
The issue has been fixed on the new release `1.3.7-SNAPSHOT`.
Thanks for your report!

**Comment by skydoves** [comment]
This issue has been released on the new stable `1.3.7`.
Thanks!
